widgetssdk/src/main/java/com/glia/widgets/di/Dependencies.java

```diff
@@ -42,9 +43,18 @@ public class Dependencies {
     private static ManagerFactory managerFactory;
     private static GliaCore gliaCore = new GliaCoreImpl();
     private static ResourceProvider resourceProvider;
+    private static StringProvider stringProvider;
 
     public static void onAppCreate(Application application) {
-        notificationManager = new NotificationManager(application);
+        resourceProvider = new ResourceProvider(application.getBaseContext());
+        stringProvider = stringKey -> {
+            // TODO IMPLEMENT NEW METHOD FROM CORE
+            String fallback = resourceProvider.getString(stringKey);
+            String key = resourceProvider.getResourceKey(stringKey);
+            // TODO IMPLEMENT NEW METHOD FROM CORE
```

Comment:
I understand that this is a draft for TODO, just want to make sure my current understanding is correct.
What needs to be done here in non-technical words is:
1. Prepare a string fallback.
2. Prepare a string key for getting string from remote source.
3. Get string from remote source. This step is missing from the draft now, right?

Reply:
Yes, that is correct. When download task gets merged in SDK, then we should be able to make a temporary release and hook the two features together.